src/frontend/scss/generic/_accessibility.scss

```diff
@@ -1,6 +1,44 @@
 // Helper class to hide elements from sighted users and have accessibility devices still read them
 // to users that need it.
-.offscreen {
+
+// Screen reader only content
+.offscreen,
```

Comment:
So do you want someone from FUN to remove the `offscreen` class and replace it with `sr-only`, or would you do it in this PR?